src/core/BufferAttribute.js

```diff
@@ -102,6 +102,16 @@ THREE.BufferAttribute.prototype = {
 
 	},
 
+	setRange: function ( index, typedArray ) {
```

Comment:
`copyAt` is like the new `set()` but with 2 offsets. It's actually only used in `webgl_interactive_buffergeometry.html`.